orchestra/tests/helpers/iterations.py

```diff
@@ -5,32 +5,31 @@
 from orchestra.utils.task_properties import get_iteration_history
 
 
-def verify_iterations(test, task_id):
+def verify_iterations(task_id):
```

Comment:
Converted this so we can use it outside of tests, not sure what best practices are for this kind of thing?